app.rb

```diff
@@ -0,0 +1,116 @@
+require_relative './student'
+require_relative './teacher'
+require_relative './book'
+require_relative './rental'
+
+class App
+  def initialize(option)
+    @option = option
+    @books = []
+    @persons = []
+    @rentals = []
+  end
+
+  # List all books.
+  def list_books
+    @books.each do |book|
+      puts "Title: #{book.title}, Author: #{book.author}"
+    end
+    @option.show_options
+  end
+
+  # List all people.
+  def list_people
+    @persons.each do |person|
+      puts "[#{person.class.name}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+    end
+    @option.show_options
+  end
+
+  # Create a person (teacher or student, not a plain Person).
+  def create_person
+    puts 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
+    person = gets.chomp
+    case person
+    when '1'
+      create_student
+    when '2'
+      create_teacher
+    else
+      puts 'Write 1 or 2'
+    end
+    @option.show_options
+  end
+
+  # Create a student
+  def create_student
+    print 'Age: '
+    age = gets.chomp
+    print 'Name: '
+    name = gets.chomp
+    print 'Has parent persmission? [Y/N]: '
+    parent_permission = gets.chomp.downcase == 'y'
+    # print 'Select classroom: '
+    # classroom = gets.chomp
```

Comment:
- [ ] Kindly ensure you remove commented code from your codebase so that your code will look clean and readable 👍🏽

Reply:
Thank you @vickymarz let me remove them!

Reply:
well resolved! ✔️